rasr/config.py

```diff
@@ -338,3 +341,32 @@ def __init__(self, string, hidden=None):
 
     def __str__(self):
         return self.string
+
+
+class WriteRasrConfigJob(Job):
+    """
+    Write a RasrConfig object into a .config file
+    """
+
+    def __init__(self, config, post_config):
+        """
+        :param RasrConfig config: RASR config part that is hashed
+        :param RasrConfig post_config: RASR config part that is not hashed
+        """
+        self.config = config
+        self.post_config = post_config
+
+        self.out_config = self.output_path("rasr.config")
+
+    def tasks(self):
+        yield Task("run", mini_task=True)
+
+    def run(self):
+        self.config._update(self.post_config)
```

Comment:
Should we make this method public?

Reply:
It should be public, but that changes a lot of files...

Reply:
I will do a new PR with this, if needed.

Reply:
I think this is fine, but feel free to do that if you like of course.